pkg/provider/bitbucketserver/types/types.go

```diff
@@ -10,9 +10,16 @@ type EventActor struct {
 }
 
 type PullRequestEvent struct {
-	Actor       EventActor
-	PullRequest bbv1.PullRequest `json:"pullRequest"`
-	Comment     bbv1.Comment     `json:"comment"`
+	Actor       bbv1.UserWithLinks `json:"actor"`
+	PullRequest bbv1.PullRequest   `json:"pullRequest"`
+	EventKey    string             `json:"eventKey"`
+
+	// Comment should be used when event is `pr:comment:added` or `pr:comment:edited`.
+	Comment bbv1.ActivityComment `json:"comment"`
+
+	// CommentParentID and PreviousComment should be used when event is `pr:comment:edited`.
+	CommentParentID string `json:"commentParentId"`
+	PreviousComment string `json:"previousComment"`
```

Comment:
can we have unit test for these newly added ( EventKey, CommentParentID and PreviousComment ) fields

Reply:
@zakisk i am okay if you want to add tests in follow up PR so that we can merge this for PAC release